tests/hw/esp-tester/mini-jag.toit

```diff
@@ -27,7 +29,12 @@ install-new-test:
     if id != containers.current:
       containers.uninstall id
 
-  network := net.open
+  network/net.Client? := null
+  for i := 0; i < NETWORK_RETRIES; i++:
```

Comment:
NETWORK-RETRIES

Reply:
https://github.com/toitlang/toit/pull/2693.
Saw it too, and then just forgot to fix it...
done.